README.md

```diff
@@ -75,12 +75,27 @@ $butterCms->fetchAuthors(['include' => 'recent_posts']);
 $butterCms->fetchCategory('category-slug');
 $butterCms->fetchCategories(['include' => 'recent_posts']);
 
+// Pages
+$butterCms->fetchPage('about', 'welcome-to-the-site');
+$pagesResponse = $butterCms->fetchPages('news', ['breaking-news' => true]);
+var_dump($pagesResponse->getMeta()['count']);
+foreach ($pagesResponse->getPages() as $page) {
+    echo $page->getSlug();
```

Comment:
How do you access one of the fields (some-field) of the page?   `page = {'slug': 'blah-blah', 'fields': { 'some-field': 'blah' } } } `

Like this?

```
$fields = $page->getFields();
$fields->getSomeField()
```

Reply:
Added at https://github.com/ButterCMS/buttercms-php/pull/4/commits/8cd9e8f8e6a1c24439112bc5f3cdf3a70d56d6cf#diff-04c6e90faac2675aa89e2176d2eec7d8R80.

The `$fields` property is essentially just an array (since those can really be anything), so you access it like an array.

If you want, I could make it work via something like this, too, for a tad more convinience:

`$page->getField('some-field')`

Would just be:

```
public function getField($fieldName, $default = null) {
    return isset($this->fields[$fieldName]) ? $this->fields[$fieldName] : $default;
}
```

This wrapper is currently set to work back to PHP 5.3 I believe, if we want to move it up to 7.0 we could also make use of `??` there, but I don't think that's worth it at the sacrifice of backwards compatibility just yet.

Reply:
Yeah adding `$page->getField('some-field')` would be great.

Ok good to know. I agree, let's stick with supporting 5.3 for now.

Reply:
Added in https://github.com/ButterCMS/buttercms-php/pull/4/commits/f34b26dd754b93127054cadc926cae3e8edfa868.